### PR TITLE
Lp 41 articles styles

### LINKED ISSUE #41: Style the news articles
**Is your feature request related to a problem? Please describe.**
The news articles look bland

**Describe the solution you'd like**
Give them more color and make them pop

### PR DESCRIPTION
**Pull request by Lap343**
## Changes
1. add color style to the box-shadow
2. On hover dims the img and says click to learn more of the article

## Purpose
Give the articles a better look

Closes #41

### CODE REVIEW

File: modern-news-app/src/components/Main.tsx [hunk 1]
```diff
@@ -120,32 +124,49 @@ const Main: React.FC<MainProps> = ({ newsData }) => {
...
+
+    // Set things like the Title color and the box shadow color
+    const mainColorCheck = (): string => {
+        switch(newsData.search){
+            case "sports": return "#FFA55C" ;
+            case "entertainment": return "#1FC8A3";
+            case "business": return "#6F81D8";
+            case "health": return "#00C0C8";
+            case "world": return "#FF9A86";
+            default : return "white";
```
Comment (krferrell) [inline modern-news-app/src/components/Main.tsx:147]: This looks too nice.

Comment (JorgeLVilla) [inline modern-news-app/src/components/Main.tsx:147]: So clean Luis!

**Review by JorgeLVilla — APPROVED** [review]
Nice job!

**Review by krferrell — APPROVED** [review]
Looking good!

**Review by connielion — APPROVED** [review]
Looking very noice!

File: modern-news-app/src/components/Main.tsx (hunk 1 above)
Comment (JorgeLVilla) [reply modern-news-app/src/components/Main.tsx:147]: Thats exactly what I told him too lol. So clean.

**Pull request merged**